examples/qwp/query/main.go

```
@@ -0,0 +1,121 @@
+
+func main() {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	client, err := qdb.NewQwpQueryClient(ctx,
+		qdb.WithQwpQueryAddress("localhost:9000"),
+	)
+	if err != nil {
+		log.Fatalf("connect: %v", err)
+	}
+	defer client.Close(ctx)
```

golangci-lint (2.11.4)

[error] Error return value of `client.Close` is not checked

(errcheck)